### PR TITLE
Process only exported expenses for Payments sync, Handle sdk exceptions

### PR DESCRIPTION
**Pull request by ashwin1111**

### CODE REVIEW

File: apps/netsuite/tasks.py [hunk 1]
```diff
@@ -963,36 +961,38 @@ def process_vendor_payment(entity_object, workspace_id, object_type):
...
 
         expense_reports = ExpenseReport.objects.filter(
-            payment_synced=False, expense_group__workspace_id=workspace_id, expense_group__fund_source='PERSONAL'
+            payment_synced=False, expense_group__workspace_id=workspace_id,
+            expense_group__fund_source='PERSONAL', expense_group__exported_at__isnull=False
         ).all()
 
-        if bills:
-            bill_entity_map = create_netsuite_payment_objects(bills, 'BILL', workspace_id)
+        with transaction.atomic():
```
Comment (ashwin1111) [inline apps/netsuite/tasks.py:977]: added transaction block here

File: apps/netsuite/tasks.py [hunk 2]
```diff
@@ -883,45 +883,43 @@ def process_vendor_payment(entity_object, workspace_id, object_type):
         }
     )
     try:
-        with transaction.atomic():
```
Comment (ashwin1111) [inline apps/netsuite/tasks.py:886]: added transaction below

File: apps/netsuite/tasks.py [hunk 3]
```diff
@@ -963,36 +961,38 @@ def process_vendor_payment(entity_object, workspace_id, object_type):
...
         fyle_credentials = FyleCredential.objects.get(workspace_id=workspace_id)
-
         fyle_connector = FyleConnector(fyle_credentials.refresh_token, workspace_id)
-
         fyle_connector.sync_reimbursements()
 
         bills = Bill.objects.filter(
-            payment_synced=False, expense_group__workspace_id=workspace_id, expense_group__fund_source='PERSONAL'
+            payment_synced=False, expense_group__workspace_id=workspace_id,
+            expense_group__fund_source='PERSONAL', expense_group__exported_at__isnull=False
```
Comment (ashwin1111) [inline apps/netsuite/tasks.py:969]: added `expense_group__exported_at__isnull=False`

File: apps/netsuite/tasks.py [hunk 4]
```diff
@@ -963,36 +961,38 @@ def process_vendor_payment(entity_object, workspace_id, object_type):
...
         ).all()
 
         expense_reports = ExpenseReport.objects.filter(
-            payment_synced=False, expense_group__workspace_id=workspace_id, expense_group__fund_source='PERSONAL'
+            payment_synced=False, expense_group__workspace_id=workspace_id,
+            expense_group__fund_source='PERSONAL', expense_group__exported_at__isnull=False
```
Comment (ashwin1111) [inline apps/netsuite/tasks.py:974]: added `expense_group__exported_at__isnull=False`

**Review by Sravanksk — APPROVED** [review]

**Pull request merged**